CRM/Utils/System/Drupal.php

```diff
@@ -522,8 +522,8 @@ public function loadBootStrap($params = [], $loadUser = TRUE, $throwError = TRUE
     $uid = $params['uid'] ?? NULL;
     if (!$uid) {
       //load user, we need to check drupal permissions.
-      $name = CRM_Utils_Array::value('name', $params, FALSE) ? $params['name'] : trim(CRM_Utils_Array::value('name', $_REQUEST));
-      $pass = CRM_Utils_Array::value('pass', $params, FALSE) ? $params['pass'] : trim(CRM_Utils_Array::value('pass', $_REQUEST));
+      $name = !empty($params['name']) ? $params['name'] : trim($_REQUEST['name'] ?? '');
```

Comment:
I think when I retire I will legally change my name to `0`, or "Dave " (note the `&nbsp;` at the end), and then entertain myself by getting CSR's to enter my info into their systems.